hanfor/lib_core/data.py

```diff
@@ -323,12 +323,14 @@ def get_tag_name_comment_dict(self) -> dict[str, str]:
 
 @DatabaseFieldType()
 @DatabaseField("id", int)
+@DatabaseField("order", int)
```

Comment:
i think the databse field should have a default value for being added to old databasese.